docs/xxjitserveraotcachedir.md

```diff
@@ -0,0 +1,51 @@
+<!--
+* Copyright (c) 2017, 2022 IBM Corp. and others
+*
+* This program and the accompanying materials are made
+* available under the terms of the Eclipse Public License 2.0
+* which accompanies this distribution and is available at
+* https://www.eclipse.org/legal/epl-2.0/ or the Apache
+* License, Version 2.0 which accompanies this distribution and
+* is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* This Source Code may also be made available under the
+* following Secondary Licenses when the conditions for such
+* availability set forth in the Eclipse Public License, v. 2.0
+* are satisfied: GNU General Public License, version 2 with
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] http://openjdk.java.net/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# -XX:JITServerAOTCacheDir
+
+**(Linux&reg; only)**
+
+This option specifies the directory to be used for saving and loading JITServer AOT cache files.
+
+## Syntax
+
+        -XX:JITServerAOTCacheDir=<directory>
+
+ If the directory name is not specified (`-XX:JITServerAOTCacheDir=`), the cache files are saved in the current working directory of the JITServer server.
+
+## Explanation
+
+ A JITServer instance can have several AOT caches, each with its own name. To allow reusing of the AOT caches by the later JITServer instances, you can use the [`-XX:+JITServerAOTCachePersistence`](xxjitserveraotcachepersistence.md) option. With this option enabled, JITServer server can periodically save its AOT caches to files and load caches from these files the first time a client requests a particular cache.
+
+ You can specify the directory for saving the AOT cache files with the `-XX:JITServerAOTCacheDir=<directory>` option. When the server receives the location of the requested AOT cache file through the `-XX:JITServerAOTCacheDir` option and a request for a specific file, if that cache does not exist in-memory, the server searches the specified cache directory for a file with that name and loads it, if available.
```

Comment:
The client does not ask for a filename, but rather for a cache name. The server takes this name and the directory from the `-XX:JITServerAOTCacheDir=<directory>` options and builds a corresponding cache filename.

"..and a request for a specific file, if that cache does not exist in-memory, the server searches the specified cache directory for a file with that name and loads it, if available."
-->
" and a request for a specific cache name, if that cache does not exist in-memory, the server searches the specified cache directory for file matching the requested name and loads it, if available."